web/lib/HabitatCommunityPreviewCreator.js

```diff
@@ -14,43 +14,89 @@ import { COMMON_STYLESHEET } from './component.js';
 const TEMPLATE = document.createElement('template');
 TEMPLATE.innerHTML = `
 <style>
-.communityBox {
+:host {
+  width:100%;
+}
+button, .button, button *, .button * {
+  background-color: var(--color-bg-button);
+}
+#create-community-box {
+  border-radius: 2em;
+  background-color: var(--color-box);
+  border: 1px solid var(--color-bg-invert);
+  width:100%;
+  margin:5em 0 0 0;
+  padding:3em;
+}
+canvas {
+  margin: .5em 0;
+  width: 100%;
+  min-width:20ch;
+  border: 1px solid var(--color-bg-invert);
   border-radius: 2em;
-  background-color: var(--color-accent-grey);
   cursor: pointer;
 }
-.communityBox canvas {
-  width: 40ch;
-  max-width: 100%;
+canvas.editor {
+  cursor: all-scroll;
+}
+#create-community-box input, textarea {
+  margin: .5em 0;
+  color: var(--color-text);
   border-radius: 2em;
+  border: 1px solid var(--color-accent-grey);
+  background-color: var(--color-bg);
+  min-width:14ch;
+  width:100%;
+  font-weight: 300;
 }
-.communityBox input {
-  border: none;
-  border-bottom: 1px solid var(--color-bg-invert);
+input::placeholder, textarea::placeholder {
+  color: var(--color-grey);
+}
+habitat-verc-creator {
+  width:100%;
+  display:none;
+}
+habitat-verc-creator.active {
+  display:block;
 }
 </style>
-<div class='communityBox'>
-  <space></space>
-  <div id='input' class='flex col'>
-    <input id='title' placeholder='Name of Community'>
-    <input id='details' placeholder='Short Description about the Community'>
-    <input id='token' placeholder='Governance Token' list='tokenlistv2'>
+<div id='create-community-box'>
+  <div class='left' style='margin-bottom: 2em;'>
+    <p class='l'><span><emoji-seedling></emoji-seedling><span> Create a Community</span></span></p>
   </div>
-  <space></space>
-  <input style='display:none;' id='file' type='file' accept='image/*'>
-  <p></p>
-  <label style='padding:1em;'>
-    Tap on the area below to add a community banner.
-    <br>
-    Aspect ratio is 2:1. i.e 1200x600
-  </label>
-  <canvas></canvas>
-  <div>
-    <button id='create'>Create</button>
+
+  <div class='flex row between' style='align-items:flex-start;flex-wrap:wrap;gap:3em;'>
+    <div id='input' class='flex col center evenly' style='width:auto; flex:1 0 0;'>
+      <input style='width:100%;height:1em;justify-content:center;' id='title' placeholder='Name of Community'>
```

Comment:
consider moving size/positioning rules into a `<style>` block. It becomes much easier to read/change later